pubspec.yaml

```diff
@@ -55,6 +55,7 @@ dependencies:
   shared_preferences: ^2.2.0
   sprintf: ^7.0.0
   sticky_headers: ^0.3.0+2
+  store_redirect: ^2.0.1
```

Comment:
I am pretty sure this package is redundant and at the same time is unmaintained. You can achieve the same thing with the url_launcher